docs/data/base/pages.ts

```diff
@@ -18,7 +18,7 @@ const pages = [
     title: 'Components',
     icon: standardNavIcons.ToggleOnIcon,
     children: [
-      { pathname: '/base-ui/react-components', title: 'All components' },
+      { pathname: '/base-ui/react-all-components', title: 'All components' },
```

Comment:
Following the convention for the pages inside `/components/`

Reply:
I could also rename the page to `react-components`, but I liked more this url. Happy to iterate on this. cc @danilo-leal

Reply:
Yeah, I think I also dig "all-components" slightly more, but that's just a loose preference. 😬

Reply:
My understanding is that the `react-*` convention is mostly for SEO purposes, which probably isn't really applicable here. So I would also lean more towards `all-components` for the sake of simplicity. (Otherwise, the grammatically correct version would be `all-react-components`.)

Reply:
Alright, I will adjust the script to not consider this file as React component file.